src/App.js

```diff
@@ -17,8 +18,36 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+const numeratedTabs = tabs.map((tab, index) => ({
+  ...tab,
+  id: index,
+}));
+
+class App extends React.PureComponent {
+  state = {
+    tabsArr: numeratedTabs,
+    selectedTabContent: '',
+
+  }
+
+  onTabSelected = (tab) => {
+    this.setState({ selectedTabContent: tab.content });
+  }
```

Comment:
let's just pass content

```suggestion
  onTabSelected = (content) => {
    this.setState({ selectedTabContent: content });
  }
```